api/views/exercises.py

```diff
@@ -0,0 +1,92 @@
+from flask import Blueprint, jsonify
+from sqlalchemy import asc
+from models import Exercise, Artifact, Language
+
+DEFAULT_LIMIT = 25
+
+
+def exercise_to_dict(exercise):
+    return {
+        'name': exercise.name,
+        'language': exercise.language.value,
+        'creator': exercise.creator,
+        'topics': exercise.topics,
+        'difficulty': exercise.difficulty,
+    }
+
+
+def complete_exercise_to_dict(exercise, artifact):
+    return {
+        'name': exercise.name,
+        'language': exercise.language.value,
+        'creator': exercise.creator,
+        'topics': exercise.topics,
+        'difficulty': exercise.difficulty,
+        'description': exercise.description,
+        'readme': artifact.readme,
+        'solution': artifact.solution,
+        'starting_point': artifact.starting_point,
+        'test': artifact.test,
+        'hint': artifact.hint,
+    }
+
+
+def create_exercises_blueprint(session, request):
+    exercises = Blueprint('exercises', __name__)
+
+    @exercises.route('/', methods=['GET'])
+    def get_exercises(default_limit=DEFAULT_LIMIT):
```

Comment:
Don't add arguments to the route function if they don't reflect anything from the request. This should either be a constant (it pretty much already is, since flask will never give you a value for this argument) or configurable for the blueprint (an argument to `create_exercises_blueprint`)